discord/ext/commands/context.py

```diff
@@ -206,6 +206,17 @@ def valid(self):
     async def _get_channel(self):
         return self.channel
 
+    @property
+    def clean_prefix(self):
+        """:class:`str`The cleaned up invoke prefix. i.e. mentions are ``@name`` instead of ``<@id>``."""
```

Comment:
```suggestion
        """:class:`str`: The cleaned up invoke prefix. i.e. mentions are ``@name`` instead of ``<@id>``.

        .. versionadded:: 2.0
        """
```